Chamasma/Presentation/Sources/Presentation/Examples/Meal+Examples.swift

```diff
@@ -0,0 +1,101 @@
+//
+//  Menu+Examples.swift
+//  
+//
+//  Created by Mariam Ormotsadze on 23.04.23.
+//
+
+import Domain
+import DesignSystem
+
+// MARK: - Example
+public extension Meal {
+    enum Example {
+        
+        public static let meals = [cheesburger, chiliCheesburger, hamburger, vegetarianBurger, tofu, salad]
+        
+        static let cheesburger = Meal(
+            name: "Cheesebuger",
+            description: "Cheese, Burger, Beef, ketchup, Lettuce",
+            rating: 0.75,
+            price: 7.89,
+            image: DesignSystem.Image.burger,
+            ingredients: [],
+            isVegetarian: false,
+            isVegan: false,
+            isGlutenFree: false,
+            isPopular: true,
+            isSpicy: false,
+            allergens: [])
+        
+        static let chiliCheesburger = Meal(
+            name: "Chili Cheesebuger",
+            description: "Chili, Cheese, Burger, Beef, ketchup, Lettuce",
+            rating: 0.45,
+            price: 8.31,
+            image: DesignSystem.Image.burger,
+            ingredients: [],
+            isVegetarian: false,
+            isVegan: false,
+            isGlutenFree: false,
+            isPopular: true,
+            isSpicy: true,
+            allergens: [])
+        
+        static let hamburger = Meal(
+            name: "Hamburger",
+            description: "Burger, Beef, ketchup, Lettuce",
+            rating: 0.89,
+            price: 6.47,
+            image: DesignSystem.Image.burger,
+            ingredients: [],
+            isVegetarian: false,
+            isVegan: false,
+            isGlutenFree: false,
+            isPopular: true,
+            isSpicy: false,
+            allergens: [])
+        
+        static let vegetarianBurger = Meal(
+            name: "Vegetarian Burger",
+            description: "Burger, Tomato, Lettuce, Cheese",
+            rating: 0.14,
+            price: 4.99,
+            image: DesignSystem.Image.burger,
+            ingredients: [],
+            isVegetarian: true,
+            isVegan: false,
+            isGlutenFree: true,
+            isPopular: false,
+            isSpicy: false,
+            allergens: [])
```

Comment:
Lint does not have problems with this?

Reply:
it turns out swiftlint was not working in Presentation module, so I added it and fixed all violations

Reply:
please, re-review it